src/sentry/static/sentry/app/views/releasesV2/detail/overview/commitAuthorBreakdown.tsx

```diff
@@ -46,14 +50,22 @@ class CommitAuthorBreakdown extends AsyncComponent<Props, State> {
   }
 
   getDisplayPercent(authorCommitCount: number): string {
-    const {commitCount} = this.props;
+    const {commits} = this.state;
 
-    const calculatedPercent = round(percent(authorCommitCount, commitCount), 0);
+    const calculatedPercent = round(percent(authorCommitCount, commits.length), 0);
```

Comment:
Why not use `Math.round`?

Reply:
No reason. I usually use lodash's round when I need decimal places. I was trying how it looked with decimal places and didn't revert back to Math.round afterward... It's changed now.